compact/tracking/mpgd_outerbarrel.xml

```diff
@@ -149,12 +149,12 @@
 <comment> Layout for MPGD DIRC layers </comment>
   <layer module="MPGDOuterBarrelModule" id="0" vis="TrackerSupportVis">
     <envelope_tolerance r_min="-50*mm" r_max="0*mm" z_min="0*mm" z_max="0*mm"/>
-    <layer_material surface="inner" binning="binPhi,binZ" bins0="MPGDOuterBarrelModule_count" bins1="100" />
+    <layer_material surface="outer" binning="binPhi,binZ" bins0="MPGDOuterBarrelModule_count" bins1="100" />
     <rphi_layout
           phi_tilt="0"
           nphi="MPGDOuterBarrelModule_count"
           phi0="0"
-          rc="MPGDOuterBarrelModule_rmin"
+          rc="MPGDOuterBarrelModule_rmin + 0.5*cm"
```

Comment:
Is this 0.5cm something we should turn into a named parameter?

Reply:
Yup, changed